client/js/components/GameListItem.vue

```diff
@@ -1,15 +1,15 @@
 <template>
   <div>
     <v-row class="list-item" data-cy="game-list-item">
-      <v-col sm="2" lg="3">
+      <v-col sm="2" lg="3" v-if="!$vuetify.breakpoint.mdAndDown">
```

Comment:
Added breakpoint so the image only shows on desktop version of screen.

Reply:
Would it make sense to use `$vuetify.breakpoint.lgAndUp` here instead?

Reply:
I agree, and was thinking about making this change as well.